demoland_engine/predictors.py

```diff
@@ -71,6 +63,14 @@ def get_indicators(df, mode="walk", random_seed=None):
     DataFrame
         DataFrame containing the resulting indicators
     """
+    matrix = FILEVAULT["matrix"]
+    aq_model = FILEVAULT["aq_model"]
+    hp_model = FILEVAULT["hp_model"]
+    accessibility = FILEVAULT["accessibility"]
+
+    air_quality_predictor = Model(matrix, aq_model)
+    house_price_predictor = Model(matrix, hp_model)
+
```

Comment:
I moved these calls inside the respective functions because otherwise they are kept in memory, even when FILEVAULT is refreshed by change_data().

In principle if the module is imported every time a call to the API is made, then this isn't needed. (e.g. this is the case with pyodide) It is only an error if there is some kind of persistence, e.g. when running API locally with uvicorn.

In any case I think it's better to do this to ensure consistency with the global state.